packages/importer/src/utils/lang.ts

```diff
@@ -1,7 +1,32 @@
+import { XataFile } from '@xata.io/client';
+
 export function isDefined<T>(value: T | undefined | null): value is T {
   return value !== undefined && value !== null;
 }
 
 export function isObject<T>(value: T | unknown): value is T {
   return isDefined(value) && typeof value === 'object';
 }
+
+function notEmpty<T>(value: T | null | undefined): value is T {
+  return value !== null && value !== undefined;
+}
+
+export function compact<T>(arr: Array<T | null | undefined>): T[] {
+  return arr.filter(notEmpty);
+}
+
+export function isXataFile(value: unknown): value is XataFile {
+  // TODO fix this predicate
+  return isObject(value) && (value as any).mediaType !== undefined;
```

Comment:
`value instanceof XataFile` was not returning true in situations where it should have